generate-passphrase

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+LENGTH=32
+if [ ! -z "$1" ]; then
+  LENGTH=$1
+fi
+cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9!@#$%^&*()_+?><~;' | fold -w $LENGTH | head -n 1
```

Comment:
@rogeruiz You might find this doesn't work too well in OSX. At least, not for me in Yosemite.. However, I have a version which does work..

```
cat /dev/urandom | LC_ALL=C tr -dc 'a-zA-Z0-9!@#\$%^&\*\(\)\+=_' | head -c $LENGTH
```